controllers/sandbox_controller.go

```diff
@@ -170,10 +248,26 @@ func (r *SandboxReconciler) reconcilePod(ctx context.Context, sandbox *sandboxv1
 		log.Info("Found Pod", "Pod.Namespace", pod.Namespace, "Pod.Name", pod.Name)
 		// TODO - Do we enfore (change) spec if a pod exists ?
 		// r.Patch(ctx, pod, client.Apply, client.ForceOwnership, client.FieldOwner("sandbox-controller"))
-		return pod, nil
+		readyCondition.Message = "Pod exists with phase: " + string(pod.Status.Phase)
+
+		// Check if pod Ready condition is true
+		if pod.Status.Phase == corev1.PodRunning {
```

Comment:
This hasn't fully addressed my feedback yet. Would you move this part to `computeReadyCondition`? This makes it clear which conditions we care about in `computeReadyCondition` when it checks whether conditions are true. Otherwise the logic is hidden here.

Reply:
mis-understood your comment. fixed now.